labs-ios-starter/CityNetworkClient.swift

```diff
@@ -50,4 +50,63 @@ class CityNetworkClient {
             completion(.success(true))
         }.resume()
     }
+    
+//    // fetch rent
+//    {
+//      "msg": "average rent",
+//      "score": 4,
+//      "avg_rent": 2100
+//    }
+    func fetchAverageRent(forCity city: String, completion: @escaping (Result<Bool, NetworkError>) -> Void) {
+        
+        let averageRentForCityURL = baseDSURL.appendingPathComponent("city_scr/\(city)")
+        
+        URLSession.shared.dataTask(with: averageRentForCityURL) { (data, _, error) in
+            if let error = error {
+                print("Error \(error)")
+            }
```

Comment:
I think this should have a `completion(.failure(. noData))` after printing the error. All exit paths out of this function should call the completion.

Also add a `return` after the `completion` line. If there's an error we don't want to run the rest of the function.